src/formats.c

```diff
@@ -72,8 +72,10 @@ static void test_fmt_case(struct fmt_main *format, void *binary,
  */
 int fmt_match(const char *req_format, struct fmt_main *format, int override_disable)
 {
+#ifndef BENCH_BUILD
 	int enabled = override_disable || !((options.flags & FLG_TEST_CHK) || options.listconf) ||
 		!cfg_get_bool(SECTION_DISABLED, SUBSECTION_FORMATS, format->params.label, 0);
+#endif
```

Comment:
This could simply be moved under the existing `#ifndef` below, near the `char *pos = NULL;` line.

Reply:
Right, I'll do that. I'm so used with trying to keep differences from core branch's code at a graspable level but we can't merge that branch anymore anyway due to git-cvs having eroded to being unusable with reasonable effort.

Oh btw I have a semaphore in my head shouting we have some tiny(?) rule change from core not yet merged into Jumbo

Reply:
Right, I'll do so. I'm still used to try keeping core files close enough to CVS that I can understand the diffs but now that `git-cvs` corroded to completely unusable without unreasonable effort, it's of no use anymore.

Reply:
I see that's not even a core function btw... A whole lot of them aren't 😆

Reply:
No, wait - it is. It's the `bench.c` one that's completely new code (and an awful cludge at that)